include/ignition/gazebo/components/Model.hh

```diff
@@ -29,14 +32,64 @@ namespace gazebo
 {
 // Inline bracket to help doxygen filtering.
 inline namespace IGNITION_GAZEBO_VERSION_NAMESPACE {
+namespace serializers
+{
+  class SdfModelSerializer
```

Comment:
Is this better suited in [Serialization.hh](https://github.com/ignitionrobotics/ign-gazebo/blob/ign-gazebo4/include/ignition/gazebo/components/Serialization.hh)?

Reply:
I don't have a strong opinion about it because I see examples in both places, `Serialization.hh` and directly in the component. I think I'd go for an approach where each component is self-contained and includes its serialization if possible. `Serialization.hh` could contain any generic serialization that can be reused across many components, such as `MsgSerializer` for example.

Reply:
Sounds good to me!